app/views/articles/index.html.erb

```diff
@@ -1,10 +1,10 @@
-<h1>Articles</h1>
+<h1><%= t('article.articles') %></h1>
 
 <div>
-  <%= render 'flash_messages' %>
+  <%= render 'articles/flash_messages' %>
```

Comment:
You can move it to the application_layout.html.erb
It's the only place where you actually need to show flash messages.
Also, no need to put this partial to articles folder. You can do 'shared/articles' and use this partial in application layout for every single flash message.